admin-data-explorer-play-server/src/main/scala/net/wiringbits/webapp/utils/admin/utils/package.scala

```diff
@@ -17,6 +17,12 @@ package object utils {
     }
   }
 
+  implicit class MapStringHideExt(val data: Map[String, String]) extends AnyVal {
+    def hideData(columnsToHide: List[String]): Map[String, String] = {
```

Comment:
I'd call this `removeKeys` but I'm surprised that such a method doesn't exists in the sdtlib